boost-ext-simd: new recipe (version 4.17.6.0) (#6692) (#6693)
conan-io/conan-center-index: recipes/boost-ext-simd/all/test_package/CMakeLists.txt

cmake_minimum_required(VERSION 3.1.0)

> project(test_package)
> 
> find_package(Boost.SIMD REQUIRED)

Reviewer: Until conan 2.0, `cmake` is our "toolchain" that applies some settings.
```suggestion
project(test_package)

include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
conan_basic_setup(TARGETS)

find_package(Boost.SIMD REQUIRED)
```

> # The components from Boost.SIMD (target Boost::SIMD) won't work,
> # as it is defined in Boost.SIMD cmake package file, so we
> # could not use the Boost.SIMD components in basic find_package(Boost).
> find_package(Boost REQUIRED COMPONENTS date_time) #  COMPONENTS SIMD)

Reviewer: Please explicitly add a `FIXME`, and tell what it should be.
```suggestion
# FIXME: components from Boost.SIMD (target Boost::SIMD) don't work.
# It should be:
# find_package(BOOST REQUIRED COMPONENTS SIMD)
find_package(Boost REQUIRED COMPONENTS date_time) #  COMPONENTS SIMD)
```
